### DISCUSSION TITLE
problems with statistical graphs

### DISCUSSION THREAD

**Discussion by anonimosito** [ampache/ampache#3411] [General]
I have updated from version 5.5.1 to version 5.5.5, and in the STATISTICS section nothing is observed, I do not know what the problem is, if someone has happened the same thing or if you know how to solve it I will be very grateful.
attached screenshot
https://ibb.co/qpkZ8wB

**Comment by lachlan-00** [comment]
i did find some issues with the templates recently are you using the squashed version?

**Reply by anonimosito** [reply]
I just upgraded to version 5.5.6 from
https://github.com/ampache/ampache/releases/download/5.5.6/ampache-5.5.6_all.zip
and I have the same problem.

**Reply by lachlan-00** [reply]
i'll have a look on monday
